src/Evolution/Systems/Cce/Actions/InitializeCharacteristicEvolution.hpp

```diff
@@ -232,6 +232,8 @@ struct InitializeCharacteristicEvolution {
       const size_t vector_size =
           Spectral::Swsh::number_of_swsh_collocation_points(
               Parallel::get<Spectral::Swsh::Tags::LMax>(cache));
+      // avoid erroneous compiler warnings that the variable is unused.
```

Comment:
silence instead of avoid/evade and it's technically not incorrect/erroneous (if the pack size were 0)